### DISCUSSION TITLE
Non CF compliant units: Are these still usable?

### DISCUSSION THREAD

**Discussion by jgriesfeller** [SciTools/iris#5856] [Q&A]
Hi, 

we use iris here at the Noregian Mteorological institute for model data reading in our pyaerocom model evaluation suite (https://github.com/metno/pyaerocom)

I noticed that from iris 3.7 on it seems no longer possible to create cubes with non CF compliant units. The use case is that observations don't necessarily use CF compliant units (e.g. `ug S m-3` for and SO4 concentration or `ug C m-3` for a CO concentration). The S snd C is just scaling factor at the end.

For the latter I get now 
```
iris.fileformats._nc_load_rules.helpers._WarnComboIgnoringCfLoad: Ignoring invalid units 'ugC/m3' on netCDF variable 'concco'.
``` 
instead of 
`ug C m-3`

Sorry, this is just a crude example from our testing, I can provide a minimum working example if it helps.

The main question is if it should still be possible to use non CF compliant units in cube or not?

Thanks for your help.

**Comment by trexfeathers** [comment]
Hi @jgriesfeller, I'm actually surprised this is ever worked. We also didn't make any obvious 'candidate' changes that might have affected this behaviour in Iris 3.7.

Are you sure it was possible to load this with the units you wanted in Iris 3.6?
